lib/parseSignatureHeader.test.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const tap = require('tap')
```

Comment:
We could dry this out a teensy bit by extracting `test` from this require, and just using that.

```suggestion
const { test } = require('tap')
```